mmdet/datasets/custom.py

```diff
@@ -68,7 +68,8 @@ def __init__(self,
                  seg_prefix=None,
                  proposal_file=None,
                  test_mode=False,
-                 filter_empty_gt=True):
+                 filter_empty_gt=True,
+                 min_size=32):
```

Comment:
In which scenario CustomDataset is used directly? Or when these changes make any difference?